src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -132,7 +128,11 @@ public void exec(final URI home) throws IOException {
      * BkBasic produces headers with addresses without slashes.
      *
      * @throws IOException If some problem inside
+     * @todo #519:30min In org.takes.rq.RqHeaders.Base#map is skipping the
```

Comment:
@sebing well in this case, your change introduced the bug, better to find out what's wrong and fix it.

At worst, let's create a separate unit test that reproduces this bug specifically, then add a puzzle to fix it and re-enable the tests that were ignored (this test, and your new unit test). The idea is, we want to break as little existing functionality as possible. Based on what I see here, your changes should not have affected this test at all.